akka-stream/src/main/scala/akka/stream/javadsl/FileIO.scala

```diff
@@ -28,11 +31,11 @@ object FileIO {
    *
    * @param f The file to write to
    */
-  def toFile(f: File): javadsl.Sink[ByteString, CompletionStage[IOResult]] = toFile(f, append = false)
+  def toFile(f: File): javadsl.Sink[ByteString, CompletionStage[IOResult]] =
+    toFile(f, util.EnumSet.of(WRITE, CREATE))
```

Comment:
This converts the Set from java to scala to java. With `new Sink(scaladsl.FileIO.toFile(f).toCompletionStage())` you'd save one conversion.